lib/era_ja/conversion.rb

```diff
@@ -45,14 +47,16 @@ def to_era(format = "%o%E.%m.%d", era_names: ERA_NAME_DEFAULTS)
 
     private
     def era_year(year, era, era_names)
-      strftime(@era_format).sub(/(%J)?%E/) { format_year(year, $1) }.sub(/%o/i) { format_era(era, era_names) }
+      strftime(@era_format).sub(/(%J)?%E/) { format_year(year, $1) }.sub(/%1?o/i) { format_era(era, era_names) }
     end
 
     def format_era(era, era_names)
       if @era_format =~ /%o/
```

Comment:
I think it is better to use the `case` than using the `if`.
Because this method uses 2 `elsif`. I feel that to use a `elsif` too many.